src/core/tsi/alts/zero_copy_frame_protector/alts_zero_copy_grpc_protector.cc

```diff
@@ -249,21 +249,39 @@ static tsi_result alts_zero_copy_grpc_protector_max_frame_size(
   return TSI_OK;
 }
 
+static bool alts_zero_copy_grpc_protector_read_frame_size(
+    tsi_zero_copy_grpc_protector* self, grpc_slice_buffer* protected_slices,
+    uint32_t* frame_size) {
+  if (self == nullptr || frame_size == nullptr) return TSI_INVALID_ARGUMENT;
```

Comment:
should this function return a tsi_result instead of a bool if we return TSI_INVALID_ARGUMENT here (and to stay consistent with the rest of the API)?